subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/CheckstylePlugin.groovy

```diff
@@ -57,6 +58,7 @@ class CheckstylePlugin extends AbstractCodeQualityPlugin<Checkstyle> {
             configProperties = { extension.configProperties }
             ignoreFailures = { extension.ignoreFailures }
             showViolations = { extension.showViolations }
+            antClassName = { getAntClassForToolVersion(extension.toolVersion) }
```

Comment:
Would the simplest thing to do is just try loading the Checkstyle 5 class, then the Checkstyle 6.8+ one if that fails?  For PMD, we needed to know the version because the command line arguments were different.